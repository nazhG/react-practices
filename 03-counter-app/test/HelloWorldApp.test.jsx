import { render } from '@testing-library/react';
import HelloWorld from '../src/HelloWorldApp';

describe("componente react", () => {
  test("debe de mostrar un h1 con el texto hola mundo", () => {
    const { container } = render(<HelloWorld />);

    expect(container).toMatchSnapshot();
  });
});
