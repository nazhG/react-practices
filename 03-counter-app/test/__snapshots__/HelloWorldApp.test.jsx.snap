// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`componente react debe de mostrar un h1 con el texto hola mundo 1`] = `
<div>
  <p>
    This is a simple React app.
  </p>
</div>
`;
